jgrapht-core/src/test/java/org/jgrapht/alg/shortestpath/BaseEppsteinKShortestPathTest.java

```diff
@@ -0,0 +1,73 @@
+/*
+ * (C) Copyright 2019-2019, by Semen Chudakov and Contributors.
+ *
+ * JGraphT : a free Java graph-theory library
+ *
+ * See the CONTRIBUTORS.md file distributed with this work for additional
+ * information regarding copyright ownership.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0, or the
+ * GNU Lesser General Public License v2.1 or later
+ * which is available at
+ * http://www.gnu.org/licenses/old-licenses/lgpl-2.1-standalone.html.
+ *
+ * SPDX-License-Identifier: EPL-2.0 OR LGPL-2.1-or-later
+ */
+package org.jgrapht.alg.shortestpath;
+
+import org.jgrapht.Graph;
+import org.jgrapht.Graphs;
+import org.jgrapht.graph.DefaultWeightedEdge;
+
+/**
+ * Base test for {@link EppsteinKShortestPath} and
+ * {@link EppsteinShortestPathIterator}.
+ *
+ * @author Semen Chudakov
+ */
+public class BaseEppsteinKShortestPathTest {
```

Comment:
Is there a reason to put this into a separate class? Is it likely that this class will be extended by any other class? If not, let's try to avoid fragmentation and merge the base and test class.

Reply:
What about this one?

Reply:
I`ve merged the graphs for test cases in the test classes and removed the base class. Is that fine?